src/app/pages/blog/article-summary/article-summary.scss

```diff
@@ -29,6 +29,11 @@
 
     .article-headline {
         margin: 0;
+
+        a:not(:hover) {
```

Comment:
Correct me if I'm wrong, but is there a better way to catch this state instead of using :not?

Reply:
I did it this way because I wanted the default behavior for hover. If I specified this with just `a {}`, then I would have had to specify `a:hover {}` explicitly. This is the easier way to know I got what I wanted.